### PR TITLE
Move comment for mondodb extension

### PR DESCRIPTION
**Pull request by VincentLanglet**
https://github.com/sonata-project/dev-kit/pull/1254#issuecomment-727167235

### CODE REVIEW

**Comment by OskarStark** [comment]
Shall we indent it more?

File: templates/project/.github/workflows/test.yaml.twig [hunk 1]
```diff
@@ -67,7 +67,8 @@ jobs:
                   coverage: pcov
 {% if branch.hasService('mongodb') %}
                   tools: composer:v{{ project.composerVersion }}, pecl
-                  extensions: mongodb-1.9.0rc1{# Change to stable version when available #}
+{# Change mondodb extension version to a stable one when available #}
```
Comment (OskarStark) [inline templates/project/.github/workflows/test.yaml.twig:70]:
```suggestion
        {# Change mondodb extension version to a stable one when available #}
```

File: templates/project/.github/workflows/test.yaml.twig [hunk 2]
```diff
@@ -67,7 +67,8 @@ jobs:
                   coverage: pcov
 {% if branch.hasService('mongodb') %}
                   tools: composer:v{{ project.composerVersion }}, pecl
-                  extensions: mongodb-1.9.0rc1{# Change to stable version when available #}
+        {# Change mondodb extension version to a stable one when available #}
```
Comment (OskarStark) [inline templates/project/.github/workflows/test.yaml.twig:70]:
```suggestion
                {# Change mondodb extension version to a stable one when available #}
```

File: templates/project/.github/workflows/test.yaml.twig [hunk 3]
```diff
@@ -67,7 +67,8 @@ jobs:
                   coverage: pcov
 {% if branch.hasService('mongodb') %}
                   tools: composer:v{{ project.composerVersion }}, pecl
-                  extensions: mongodb-1.9.0rc1{# Change to stable version when available #}
+                {# Change mondodb extension version to a stable one when available #}
```
Comment (OskarStark) [inline templates/project/.github/workflows/test.yaml.twig:70]:
```suggestion
                  {# Change mondodb extension version to a stable one when available #}
```

**Comment by VincentLanglet** [comment]
@OskarStark the indent wasnt added on purpose.

Devkit update Pr are failing now

**Comment by OskarStark** [comment]
Oh sorry

**Pull request merged**